benchmarks/benchmarks/spatial.py

```diff
@@ -364,7 +366,7 @@ class XdistWeighted(Benchmark):
         [10, 20, 100],
         ['euclidean', 'minkowski', 'cityblock', 'sqeuclidean', 'cosine',
          'correlation', 'hamming', 'jaccard', 'chebyshev', 'canberra',
-         'braycurtis', 'yule', 'dice', 'kulsinski', 'rogerstanimoto',
+         'braycurtis', 'yule', 'dice', 'kulsinski1', 'rogerstanimoto',
```

Comment:
I think there is a typo
```suggestion
         'braycurtis', 'yule', 'dice', 'kulczynski1', 'rogerstanimoto',
```